Reword a few things and fix some errors in the guide

jmayntzhusen opened the pull request.

PRMerger7 commented:
@jmayntzhusen : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

PRMerger8 commented:
@jmayntzhusen : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

TerryGLee commented on docs/ide/tutorial-csharp-aspnet-core.md:
```diff
@@ -19,7 +19,7 @@ ms.workload:
 
-In this tutorial for C# development with ASP.NET Core using Visual Studio, you'll create a C# ASP.NET Core web app, make changes to it, explore some features of the IDE, and then run the app.
+In this tutorial for C# development with ASP.NET Core using Visual Studio, you will learn how you can create, make changes to and run a C# ASP.NET Core web app. You will also get to explore some features of the Visual Studio IDE.
 
```
@jmayntzhusen I must respectfully decline this change. We've got internal guidelines that we follow for voice and one of those is to use contractions. Please see [Project friendliness](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#project-friendliness) section of the [Top 10 tips for Microsoft style and voice](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice) topic in the Microsoft Style Guide for more information. Otherwise, while we appreciate another outlook on the words we've selected, we do try to not add extra words if they aren't necessary, as the first sentence is what typically shows up in search results and those are limited by a character count.

TerryGLee commented on docs/ide/tutorial-csharp-aspnet-core.md:
```diff
@@ -126,11 +126,11 @@ This solution follows the **Razor Page** design pattern. It is different than th
 
-1. Choose **About.cshtml.cs**. Then, clean up the `using` directives at the top of the file by using the following shortcut:
+1. Choose **About.cshtml.cs**. Then, clean up the `using` directives at the top of the file by using the following procedure:
 
```
@jmayntzhusen I must also respectfully decline this edit. The information is an addendum to a step that's already in a procedure, and so we would not want to name this as a procedure within a procedure.

TerryGLee commented on docs/ide/tutorial-csharp-aspnet-core.md:
```diff
@@ -126,11 +126,11 @@ This solution follows the **Razor Page** design pattern. It is different than th
 
-1. In Visual Studio, choose **About.cshtml**. Then, delete the word _changed_ and in its place, add the words _file and directory_.
+1. In Visual Studio, choose **About.cshtml**. Then, delete the word _additional_ and in its place, add the words _file and directory_.
 
```
@jmayntzhusen This is a good edit for technical accuracy; I'd like to keep this edit. :)

TerryGLee commented on docs/ide/tutorial-csharp-aspnet-core.md:
```diff
@@ -192,4 +192,4 @@ Congratulations on completing this tutorial! We hope you learned a little bit ab
 
 ## See also
 
```
@jmayntzhusen I am hesitant about making this change because the link works as is, but we can try it and see if the change passes our build verification process.

TerryGLee commented:
@jmayntzhusen Thanks for your PR to help improve our content! As you can see, I've asked for some changes; if you can make the changes that I've requested, we can approve the PR and get it processed.

PRMerger6 commented:
@jmayntzhusen : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

jmayntzhusen commented:
Hey @TerryGLee 

Sorry I was not familiar with the style guide - I simply found some of the sentences a bit confusing in their wording but I have reverted my changes now. 

The double slash will work in a URL but it is also unnecessary that is why I removed it but I can readd if you wish 😁

TerryGLee commented:
@jmayntzhusen Thanks again for your contribution! 
#sign-off
